### PR TITLE
[HUST CSE]Modify invalid links

### PR DESCRIPTION
**Pull request by arche8**
## 拉取/合并请求描述：(PR description)

[
#### 为什么提交这份PR (why to submit this PR)
README 里有许多文档对应的官网链接已经失效，点击会直接跳转主页，需要替换。

#### 你的解决方案是什么 (what is your solution)
找到真实的链接，对相应的失效链接进行替换。

#### 在什么测试环境下测试通过 (what is the test environment)
只是对链接进行替换，不影响编译运行。
]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Review by mysterywolf — APPROVED** [review]

File: bsp/imxrt/docs/IMXRT系列驱动介绍.md [hunk 1]
```diff
@@ -30,15 +30,15 @@
...
-| 3    | [soft I2C](https://www.rt-thread.org/document/site/programming-manual/device/i2c/i2c/) | 通过软件 I2C 或者硬件 I2C 收发数据                            |
-| 4    | [SPI](https://www.rt-thread.org/document/site/programming-manual/device/spi/spi/) | 通过 SPI 收发数据                                |
-| 5    | [ADC](https://www.rt-thread.org/document/site/programming-manual/device/adc/adc/) | 测量管脚上的模拟量                               |
-| 6    | [HWTIMER](https://www.rt-thread.org/document/site/programming-manual/device/hwtimer/hwtimer/) | 使用硬件定时器实现测量时间和定时执行回调函数功能 |
-| 7    | [PWM](https://www.rt-thread.org/document/site/programming-manual/device/pwm/pwm/) | 在特定的管脚输出 PWM 波形                        |
-| 8    | [RTC](https://www.rt-thread.org/document/site/programming-manual/device/rtc/rtc/) | 设置和读取时间                                   |
-| 9   | [WATCHDOG](https://www.rt-thread.org/document/site/programming-manual/device/watchdog/watchdog/) | 看门狗驱动                                       |
+| 1    | [GPIO](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/pin/pin.md) | 操作 GPIO 管脚                                   |
```
Comment (Guozhanxin) [inline bsp/imxrt/docs/IMXRT系列驱动介绍.md:33]: 这里不用加.md

Comment (arche8) [reply bsp/imxrt/docs/IMXRT系列驱动介绍.md]: 已经修改

File: bsp/renesas/docs/Introduction_to_RA_Series_Drives.md [hunk 2]
```diff
@@ -30,17 +30,17 @@ The driver libraries currently provided by RT-Thread already support multiple se
...
-| [QSPI](https://www.rt-thread.org/document/site/programming-manual/device/spi/spi/#qspi_1) | Send and receive data via SPI (1, 2, 4-wire).                   |
+| [TIMER](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/hwtimer/hwtimer) | Use hardware timers to implement the function of measuring time and timing execution callback functions |
+| [PWM](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/pwm/pwm) | Outputs a PWM waveform at a specific pin                        |
+| [RTC](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/rtc/rtc) | Set and read times                                   |
+| [WDT](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/watchdog/watchdog) | Watchdog driver                                       |
+| [QSPI](https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/spi/spi) | Send and receive data via SPI (1, 2, 4-wire).                   |
```
Comment (shandianchengzi) [inline bsp/renesas/docs/Introduction_to_RA_Series_Drives.md:43]: 缺少锚点：https://www.rt-thread.org/document/site/#/rt-thread-version/rt-thread-standard/programming-manual/device/spi/spi?id=配置 QSPI 设备

**Review by shandianchengzi — CHANGES_REQUESTED** [review]
change req

File: bsp/renesas/docs/Introduction_to_RA_Series_Drives.md (hunk 2 above)
Comment (shandianchengzi) [reply bsp/renesas/docs/Introduction_to_RA_Series_Drives.md:43]: https://github.com/RT-Thread/rt-thread/issues/7270

**Comment by arche8** [comment]
已经进行了修改

**Review by Guozhanxin — APPROVED** [review]

**Review by shandianchengzi — APPROVED** [review]

**Pull request merged**